File: Website/table.php
<?php
	class Table{

		public $ID;
		public $Store;
		public $Capacity;
		public $Availability;

		function __construct($ID,$Store,$Capacity,$Availability){
			$this->ID = $ID;
			$this->Store = $Store;
			$this->Capacity = $Capacity;			
			$this->Availability = $Availability;
		}

		public function updateAvailability($newAvailability){
			$this->Availability = $newAvailability;
		}
	}
?>

File: Android/Reserve/app/src/main/java/com/example/reserve/Table.java
package com.example.reserve;

public class Table {
    public int ID;
    public int Store;
    public int Capacity;
    public String Availability;

    public Table(int ID, int Store, int Capacity, String Availability){
        this.ID = ID;
        this.Store = Store;
        this.Capacity = Capacity;
        this.Availability = Availability;
    }

    public void updateAvailability(String newAvailability){
        this.Availability = newAvailability;
    }
}
